### DISCUSSION TITLE
Make quitting or resetting Taskbar revert the "Fully hide Dock" option

### DISCUSSION THREAD

**Discussion by hencomb** [lawand-dot-io/taskbar#72] [General]
As a test, I dragged the app to the bin. Even restarted, but the original Dock does not reappear.

This needs to be addressed and at the very least instructions provided on how to restore the Dock.

**Comment by lawand-dot-io** [comment]
@hencomb Thanks for the suggestion. I agree that this could be confusing for some users. I should add more information about this when this options is enabled. For your case, if you want to uninstall Taskbar and want to restore the Dock, you need to run Taskbar again and uncheck the "Fully hide Dock" option.

**Comment by lawand-dot-io** [comment]
I should also think about whether I can restore the Dock whenever Taskbar is uninstalled. I will look into this

**Comment by lawand-dot-io** [comment]
I also know that when the "Reset Preferences" option is used, the Dock doesn't get restored if fully hidden. I should fix this in a future version

**Comment by lawand-dot-io** [comment]
This is now fixed in the latest update of Taskbar (1.3.0) which is now released :)
